ProcessLib/BoundaryCondition/NormalTractionBoundaryCondition.h

```diff
@@ -53,9 +50,7 @@ class NormalTractionBoundaryCondition final : public BoundaryCondition
                         GlobalVector& b) override;
 
 private:
-    /// Vector of lower-dimensional elements on which the boundary condition is
-    /// defined.
-    std::vector<MeshLib::Element*> _elements;
+    MeshLib::Mesh& _bc_mesh;
```

Comment:
const ref to the mesh sufficient?
This also applies to many function signatures changed in this PR.